tests/lax_scipy_test.py

```diff
@@ -610,6 +612,19 @@ def test_spectral_dac_svd(self, linear_size, seed, dtype):
     v_eye = jnp.eye(v_unitary_delta.shape[0], dtype=dtype)
     self.assertAllClose(v_unitary_delta, v_eye, atol=eps)
 
+  @parameterized.named_parameters(jtu.cases_from_list(
+      {"testcase_name": f"_{jtu.format_shape_dtype_string((n_obs, n_codes, *n_feats), dtype)}",
+       "n_obs": n_obs, "n_codes": n_codes, "n_feats": n_feats, "dtype": dtype}
+      for n_obs in range(1, 3)
+      for n_codes in range(1, 3)
```

Comment:
We should test some input shapes that are not broadcast-compatible. every pair of `n_obs, n_codes` generated here would pass a broadcast check. Maybe `n_obs in [1, 3, 5]`, `n_codes in [2, 4]` or something like that?

Reply:
Ah, I forget that `list(range(1, 3)) == [1, 2]` not `[1, 2, 3]`, sorry, I will fix it.

Reply:
Done.